app/bundles/EmailBundle/EventListener/LeadSubscriber.php

```diff
@@ -102,13 +103,16 @@ protected function addEmailEvents(LeadTimelineEvent $event, $state)
                 } else {
                     $eventName = $label;
                 }
-
+                if ('failed' == $state) {
+                    $state = 'sent';
```

Comment:
this was changed to get the correct date column later in the code dateSent in the UI it reads as Falied